.azure-pipelines/cpu-unit-test.yml

```diff
@@ -16,6 +16,8 @@ steps:
       echo "##vso[task.prependpath]$HOME/.local/bin"
     displayName: Export path
   - script: |
+      apt update
+      apt install -y libopenmpi-dev
```

Comment:
Does you code run CPU UT?

Reply:
FYI. The cpu-unit-test failure https://msrasrg.visualstudio.com/SuperBenchmark/_build/results?buildId=38847&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=5caf77c8-9b10-50ef-b5c7-ca89c63e1c86&l=473

Reply:
Fixed, thanks, @jeseszhang1010  and @cp5555.
Also, could I get access to the pipeline for cpu-ut and cuda-ut.